app/templates/user-profile.html

```diff
@@ -34,30 +35,60 @@
 
 {% endblock %}
 
+{% from "_macros.html" import get_user_avatar_url %}
+
 {% block content %}
 
+<section class="b-profile-header">
+    <div class="e-bg" style="background-image: url('{{ get_user_avatar_url(user) }}')"></div> 
+    <div class="e-left">
+        <img src="{{ get_user_avatar_url(user) }}" alt="" class="e-picture">
+        <span class="e-edit material-icons">edit</span>
+    </div>
+    <div class="e-right">
+        <h2 class="e-name">{{ user.full_name }}</h2>
+        <h3 class="e-job">{{ user.position }}</h3>
+
+        <ul class="e-info">
+           {% if user.organization %}
+             <li><span class="material-icons">business</span>{{ user.organization }}</li> 
+           {% endif %}
+
+           {% if user.expertise_domain_names %}
+             {% for domain in user.expertise_domain_names %}
+               <li><span class="material-icons">school</span>{{ domain }}</li>
+             {% endfor %}
+           {% endif %}
+
+           {% if user.full_location %}
+           <li><span class="material-icons">location_on</span>{{ user.full_location }}</li> 
+           {% endif %}
+
+           {% if user.locales %}
+             {% for locale in user.locales %}
+               {# TODO: Replace 'en' below w/ current locale. #}
+               <li>{{ locale.get_display_name(locale='en') }}</li>
```

Comment:
I'm not sure if there's a better way to get the locale name in the current user's locale, rather than in the locale's locale. For instance, if I do `locale.display_name` (which is what the old profile code did) and locale is Arabic, then it outputs "Arabic" *in Arabic*, not in the current user's locale.

Aside from that, I'm also not sure how to get the current user's locale, so I've left that as a TODO.